.github/workflows/continous-integration.yml

```diff
@@ -148,7 +148,7 @@ jobs:
         if: needs.changes.outputs.backend == 'true'
         run: |
           sudo apt-get -y install libpq-dev
-          make install-full
+          make install-full || make install-full
```

Comment:
Please add comments to document why we are repeating dependency install for future reference.

Reply:
I was just experimenting with the failed tests in CI, I've removed these changes from this PR